uaclient/cli.py

```diff
@@ -500,6 +500,14 @@ def wrapper(*args, **kwargs):
                 logging.exception("KeyboardInterrupt")
             print("Interrupt received; exiting.", file=sys.stderr)
             sys.exit(1)
+        except util.UrlError as exc:
+            with util.disable_log_to_console():
+                logging.exception(exc)
```

Comment:
The argument here should be the message we want in the log attached to the traceback; is `exc` really what we want (it could be!)?

Reply:
Ok, let's log the full traceback for this which I think is not too helpful, and follow up with a more useful log message in #868

Reply:
The call to `logging.exception` is what logs the traceback. You haven't addressed my question about what message should be attached to that traceback.

Reply:
I misunderstood your intent given our earlier conversation at standup.  What I *think* for the moment (before #868 is addressed) is to log the action str representation of the exception instance which is what we are currently doing with logging.exception(exc). 
What I propose we do in #868 is no longer log the exception traceback at all, but maybe MESSAGE_CONNECTIVITY_ERROR: <str(exc)> + str(exc.url).

Reply:
Sounds good, thanks!